insights/internal/uploader/upload.go

```diff
@@ -134,7 +134,7 @@ func (um Uploader) BackoffUpload(source string, force bool) (err error) {
 		wait := time.Duration(rand.Int63n(int64(max(exp, 1)))) // #nosec:G404 We don't need cryptographic randomness.
 
 		attempts++
-		if attempts > um.maxAttempts {
+		if attempts > int(um.maxAttempts) {
```

Comment:
I'm decided on that one: 

The type cast could overflow if, in the future, we decide to make  `maxAttempts` an int64 (don't see why we would do it, but I can't see the future anyways  :p ). We could make
`var attempts int32` to make it explicitely the same type as `maxAttempts` (the linter would probably complain about the unnecessary initialization to 0 if we appended ` = 0`) to avoid this type cast. That seems to hurt readability of that line a little bit, but would otherwise flag that the type is important for some reason. I don't know. WDYT?